File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/TreeView/Features/SelectedImagePage.vb
Namespace CompanionKit.Controls.TreeView.Features

	Public Class SelectedImagePage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

            ' Sets image list that is used for TreeView node
            Me.mobjTreeView.ImageList = New ImageList
            Me.mobjTreeView.ImageList.Images.Add(New ImageResourceHandle("16x16.SelectedTreeNode.gif"))
            Me.mobjTreeView.ImageList.Images.Add(New ImageResourceHandle("16x16.DefaultTreeNode.gif"))

            ' Sets image index for tree node of normal and selected node.
            Me.mobjTreeView.ImageIndex = 1
            Me.mobjTreeView.SelectedImageIndex = 0

		End Sub
	End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/TreeView/Features/SelectedImagePage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.TreeView.Features
{
    public partial class SelectedImagePage : UserControl
    {
        public SelectedImagePage()
        {
            InitializeComponent();            
            
            // Sets image list that is used for TreeView node
            this.mobjTreeView.ImageList = new ImageList();
            this.mobjTreeView.ImageList.Images.Add(new global::Gizmox.WebGUI.Common.Resources.ImageResourceHandle("16x16.SelectedTreeNode.gif"));
            this.mobjTreeView.ImageList.Images.Add(new global::Gizmox.WebGUI.Common.Resources.ImageResourceHandle("16x16.DefaultTreeNode.gif"));
            
            // Sets image index for tree node of normal and selected node.
            this.mobjTreeView.ImageIndex = 1;
            this.mobjTreeView.SelectedImageIndex = 0;
        }
    }
}